src/py42/services/cases.py

```diff
@@ -0,0 +1,194 @@
+from collections import OrderedDict
+
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.settings import items_per_page
+
+
+class CasesService(BaseService):
+    """`Rest documentation https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases`__ ."""
```

Comment:
Also, the link is not correct.. It would belike:

```
`REST Documentation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__
```

Reply:
Updated.